core/src/main/java/hudson/Util.java

```diff
@@ -1187,7 +1187,21 @@ public static void copyFile(@Nonnull File src, @Nonnull File dst) throws BuildEx
         cp.setTofile(dst);
         cp.setFile(src);
         cp.setOverwrite(true);
+        cp.setPreserveLastModified(true);
```

Comment:
I would consider removing the use of the Ant API from this method entirely and doing a standard NIO copy with the necessary flags to keep whatever attributes are desired.

Reply:
I like this idea. The any copy here seemed strange as it is.

Reply:
Yeah my guess is that the Any copy is here for legacy reasons and can be replaced. Probably a good idea to do a `git blame` to check for any relevant historical context just in case.

Reply:
The relevant historical context appears to be that 12 years ago when the function was last modified, Java 7 didn't exist.